tvtk/pyface/ui/qt4/decorated_scene.py

```diff
@@ -147,14 +148,26 @@ def _save_snapshot(self):
         to an image.  Note that the extension of the filename
         determines what image type is saved.  The default is PNG.
         """
+
         if self._panel is not None:
-            wildcard = "PNG images (*.png)|*.png|Determine by extension (*.*)|*.*"
+            extensions = ['*.png', '*.jpg', '*.tiff', '*.bmp', '*.ps',
+                          '*.eps', '*.pdf', '*.tex', '*.rib', '*.wrl',
+                          '*.oogl', '*.vrml', '*.obj', '*.iv']
```

Comment:
Turns out the Mayavi also supports exporting to .pov (Povray) and also .x3d (X3D) files so could you please add those too?  You might need to add that to the main menu also while you are at it.